src/Paystack.php

```diff
@@ -278,4 +278,264 @@ private function getData()
         return $this->getResponse()['data'];
     }
 
-}
\ No newline at end of file
+    /**
+     * Create a plan
+     */
+    public function createPlan(){
+
+        $data = [
+            "name" => request()->name,
+            "description" => request()->desc,
+            "amount" => intval(request()->amount),
+            "interval" => request()->interval,
+            "send_invoices" => request()->send_invoices,
+            "send_sms" => request()->send_sms,
+            "currency" => request()->currency,
+        ];
+
+        $this->setRequestOptions();
+
+        $this->response = $this->setGetResponse("/plan", $data);
+
+    }
+
+    /**
+     * Fetch any plan based on its plan id or code
+     * @param $plan_code
+     * @return array
+     */
+    public function fetchPlan($plan_code){
+
+        $this->setRequestOptions();
+
+        return $this->setGetResponse('/plan/' . $plan_code, [])->getResponse();
+
+    }
+
+    /**
+     * Update any plan's details based on its id or code
+     * @param $plan_code
+     * @return array
+     */
+    public function updatePlan($plan_code){
+
+        $data = [
+            "name" => request()->name,
+            "description" => request()->desc,
+            "amount" => intval(request()->amount),
+            "interval" => request()->interval,
+            "send_invoices" => request()->send_invoices,
+            "send_sms" => request()->send_sms,
+            "currency" => request()->currency,
+        ];
+
+        $this->setRequestOptions();
+
+        return $this->setGetResponse('/plan/' . $plan_code, $data)->getResponse();
+
+    }
+
+    /**
+     * Create a customer
+     * @return array
+     */
+    public function createCustomer(){
+
+        $data = [
+            "email" => request()->email,
+            "first_name" => request()->fname,
+            "last_name" => request()->lname,
+            "phone" => request()->phone,
+            "metadata" => request()->addtional_info /* key => value pairs array */
+
+        ];
+
+        $this->setRequestOptions();
+
+        $this->response = $this->setGetResponse('/customer', $data);
+
```

Comment:
Please explain how you are using a get method. Since you are trying to create a customer. shouldn't it be a post method?